clang/test/Preprocessor/wasm-target-features.c

```diff
@@ -166,6 +166,8 @@
 // GENERIC-INCLUDE-DAG: #define __wasm_mutable_globals__ 1{{$}}
 // GENERIC-INCLUDE-DAG: #define __wasm_reference_types__ 1{{$}}
 // GENERIC-INCLUDE-DAG: #define __wasm_sign_ext__ 1{{$}}
+// GENERIC-INCLUDE-DAG: #define __wasm_nontrapping_fptoint__ 1{{$}}
+// GENERIC-INCLUDE-DAG: #define __wasm_bulk_memory__ 1{{$}}
```

Comment:
Nit: Probably my OCD, but I've trying to maintain the sorted order of the features... Can we move these lines into the sorted order?

Reply:
Done